lntest/itest/lnd_test.go

```diff
@@ -998,6 +983,67 @@ func (c commitType) Args() []string {
 	return nil
 }
 
+// calcStaticFee calculates appropriate fees for commitment transactions.  This
+// function provides a simple way to allow test balance assertions to take fee
+// calculations into account.
+func (c commitType) calcStaticFee(numHTLCs int) btcutil.Amount {
+	const (
+		htlcWeight = input.HTLCWeight
+		feePerKw   = btcutil.Amount(50 * 1000 / 4)
+	)
+
+	commitWeight := btcutil.Amount(input.CommitWeight)
+	anchors := btcutil.Amount(0)
+
+	// The anchor commitment type is slightly heavier, and we must also add
+	// the value of the two anchors to the resulting fee the initiator
+	// pays.
+	if c == commitTypeAnchors {
+		commitWeight = btcutil.Amount(input.AnchorCommitWeight)
+		anchors = 2 * btcutil.Amount(anchorSize)
+	}
+
+	return feePerKw*(commitWeight+
```

Comment:
If we make this `chainfee.SatPerKWeight`, then we can just call `FeeForWeight` here.

Reply:
fixed